http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/DummyNodepoolConfig.scala

```diff
@@ -0,0 +1,6 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+import org.broadinstitute.dsde.workbench.google2.MachineTypeName
+import org.broadinstitute.dsde.workbench.leonardo.NumNodes
+
+case class DummyNodepoolConfig(machineType: MachineTypeName, numNodes: NumNodes, autoscalingEnabled: Boolean)
```

Comment:
Yes, but I wanted to leave it configurable via a `.conf` in case that needs to change. The alternative is hard-coding it, so I'd prefer to have the configurability here